pkgs/racket-doc/json/json.scrbl

```diff
@@ -123,9 +123,10 @@ the @rfc for more information about JSON.
   remains. Like @racket[read], the function leaves all remaining
   characters in the port so that a second call can retrieve the
   remaining JSON input(s). If the JSON inputs aren't delimited per se
-  (true, false, null), they  must be separated by whitespace from the
-  following JSON input. 
-  
+  (true, false, null), they must be separated by whitespace from the
+  following JSON input. Raises @racket[exn:fail:read?] if @racket[in] is not
```

Comment:
The documentation normally refers to exceptions by the name without `?`.

Reply:
Ah, OK, thanks. Fixed.